src/scenes/end.js

```diff
@@ -1,112 +1,114 @@
 import strings from "../assets/strings.js";
 import options from "../options.js";
 
-export default class EndScene extends Phaser.Scene {
```

Comment:
These diffs are a little ugly, but if you view without whitespace, you can see they're mostly indentation.